plugins/moveToTag.py

```diff
@@ -0,0 +1,67 @@
+import MAPI
+import re
+from MAPI.Util import *
+from MAPI.Time import *
+from MAPI.Struct import *
+
+import pprint
+
+from plugintemplates import *
+
+class moveToTag(IMapiDAgentPlugin):
+
+    createFolder = 0
```

Comment:
I would use False or True here instead of 0 or 1. Booleans make the code more readable